### PR TITLE
Optimize the implantation of EXPLAIN ANALYZE

### PR DESCRIPTION
**Pull request by ycycse**
In this PR, some inappropriate implementations were optimized:
1. Error handling of Symbol
2. Add a new ExplainAnalyzeNode for table model
3. Clean the method in Statement.class
4. Set default region for ExplainAnalyzeNode

### CODE REVIEW

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/planner/distribute/TableDistributedPlanGenerator.java [hunk 1]
```diff
@@ -142,6 +140,11 @@ public List<PlanNode> visitPlan(
     return Collections.singletonList(newNode);
   }
 
+  @Override
+  public List<PlanNode> visitExplainAnalyze(ExplainAnalyzeNode node, PlanContext context) {
+    return genResult(node, context);
```
Comment (JackieTien97) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/planner/distribute/TableDistributedPlanGenerator.java:145]: call genResult for child

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/ExplainAnalyzeNode.java [hunk 2]
```diff
@@ -74,6 +68,8 @@ public PlanNode replaceChildren(List<PlanNode> newChildren) {
   protected void serializeAttributes(ByteBuffer byteBuffer) {
     PlanNodeType.EXPLAIN_ANALYZE.serialize(byteBuffer);
     ReadWriteIOUtils.write(verbose, byteBuffer);
+    ReadWriteIOUtils.write(queryId, byteBuffer);
+    ReadWriteIOUtils.write(timeout, byteBuffer);
```
Comment (JackieTien97) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/ExplainAnalyzeNode.java:72]: throw exception for these serde methods and add comments for that.

**Review by JackieTien97 — CHANGES_REQUESTED** [review]

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/planner/distribute/TableDistributedPlanGenerator.java (hunk 1 above)
Comment (ycycse) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/planner/distribute/TableDistributedPlanGenerator.java]: done

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/ExplainAnalyzeNode.java (hunk 2 above)
Comment (ycycse) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/ExplainAnalyzeNode.java]: done

**Review by JackieTien97 — APPROVED** [review]

**Pull request merged**